sklearn/metrics/pairwise.py

```diff
@@ -1678,8 +1678,11 @@ def pairwise_kernels(X, Y=None, metric="linear", filter_params=False,
         If metric is "precomputed", X is assumed to be a kernel matrix.
         Alternatively, if metric is a callable function, it is called on each
         pair of instances (rows) and the resulting value recorded. The callable
-        should take two arrays from X as input and return a value indicating
-        the distance between them.
+        should take two rows from X as input and return a value indicating
+        the distance between them. This means that callables from
+        ``sklearn.metrics.pairwise`` are not allowed, as they operate on
```

Comment:
Single backticks for linking?

Reply:
linking the module? what would that link?

Reply:
to the API ref

Reply:
I changed it, but will be surprised if it works.

Reply:
If by surprised you mean that the single backticks have way too much magic to them, I 100% agree ;)

Reply:
I didn't realize that was a valid target but I guess there is a separate section in the API docs so it makes sense.